suites/quincy/rbd/tier-2_rbd_mirror_image_operations.yaml

```diff
@@ -168,7 +168,12 @@ tests:
       clusters:
         ceph-rbd1:
           config:
-            imagesize: 2G
+            rep_pool_config:
```

Comment:
can you please make this config change for pacific as well?

Reply:
We can do it if we face any issues in pacific runs. Otherwise I feel it will be a redundant work considering our priorities right now. This PR is just concentrated on TFA fix and not on improving RBD code I feel.
Please let me know your thoughts.